templates_cspm_eventbridge/OrgFullInstall.yaml

```diff
@@ -142,6 +142,12 @@ Resources:
               - Effect: Allow
                 Action: 'events:PutEvents'
                 Resource: !Sub ${EventBusARN}
+              - Effect: Allow
+                Action:
+                  - "events:DescribeRule"
+                  - "events:ListTargetsByRule"
+                Resource:
+                  - !Sub arn:aws:events:*:*:rule/${EventBridgeRoleName}
```

Comment:
It's probably the same value, but is this correct? rule vs role?

Reply:
Yeah, for whatever reason we have a single variable used for both rule name and role name. And this is across all installs.